velox/docs/functions/spark/math.rst

```diff
@@ -152,6 +152,13 @@ Mathematical Functions
         SELECT CAST(1 as DECIMAL(20, 3)) * CAST(0 as DECIMAL(20, 3)); -- decimal 0.000000
         SELECT CAST(201e-38 as DECIMAL(38, 38)) * CAST(301e-38 as DECIMAL(38, 38)); -- decimal 0.0000000000000000000000000000000000000
 
+.. spark:function:: normalize_nan(x) -> [same as x]
+
+    Normalize NaNs. This function guarantees the generation of a single hash 
+    value for different NaNs by the hash functions. Spark used this function to 
```

Comment:
I wonder where would different kinds of NaN values come from?Do we expect to read these from storage or produced by Velox computation?